assets/js/vue-apps/form-components/address-line.vue

```diff
@@ -0,0 +1,33 @@
+<script>
+export default {
+    props: ['name', 'label', 'isRequired', 'value', 'size'],
+    computed: {
+        required() {
+            return this.isRequired === true;
+        }
+    }
+};
+</script>
+
+<template>
+    <div class="ff-address__field">
```

Comment:
Minor nit,  but having a child `ff-address__field` as the root class suggests this should be it's own top-level class name, or move the class to `address-lookup.vue`  (doesn't need changing now though)